examples/map/map_segment.py

```diff
@@ -0,0 +1,119 @@
+"""
+=======================================================
+Segmenting a Map based on transformation of coordinates
+=======================================================
+
+A demostration creating a segment of a particular map from
+transformed coordinates.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+from reproject import reproject_interp
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.wcs import WCS
+
+import sunpy.map
+from sunpy.coordinates import Helioprojective, NorthOffsetFrame, RotatedSunFrame, transform_with_sun_center
+from sunpy.data.sample import AIA_171_ROLL_IMAGE
```

Comment:
You shouldn't use roll image (just use `AIA_171_IMAGE`) because your example is killing RTD because of memory usage. The roll image is a full size 4k x 4k image and the others are downsampled.

Reply:
Ah, I see. I shall fix this..